WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomAmounts/PriceFieldFormatter.swift

```diff
@@ -15,11 +15,7 @@ class PriceFieldFormatter {
     /// Current amount converted to Decimal.
     ///
     var amountDecimal: Decimal? {
-        guard let amountDecimal = currencyFormatter.convertToDecimal(amount) else {
-            return nil
-        }
-
-        return amountDecimal as Decimal
+        return decimalFormatter.number(from: amount)?.decimalValue
```

Comment:
Question (not for this PR): Should we allow this property to be of an optional type? As we're moving the responsibility of handling nil-values to the components who consume the price formatter, rather than dealing with issues directly here on an centralised manner.

Reply:
I don't have a quick answer to this. Looking specifically at `PriceFieldFormatter`,  `amountDecimal` being optional probably makes sense, since `amount` can also be empty.

The other option I imagine is to return `0` and maybe `log` the failure? I'll check how this method is used. If all the callers just turn this value into `0`, we could also centralize it in `PriceFieldFormatter`.

Reply:
I removed optionality from `PriceFieldFormatter` `decimalValue`